widget/playground/src/containers/StyleLayout/StyleLayout.General.tsx

```diff
@@ -60,13 +75,16 @@ export function General() {
     [secondaryBorderRadius]
   );
 
-  const handleFontClick = useCallback(() => {
-    setShowModal(true);
-  }, []);
-
   return (
     <>
       <GeneralContainer>
+        <ItemPicker
+          onClick={() => setModalState(ModalState.DEFAULT_LANGUAGE)}
+          value={{ label: LANGUAGES.find((l) => l.value === language)?.name }}
```

Comment:
In Figam I saw a logo beside the title. in the ItemPicker component we have a props called `hasLogo`. I think we can modify it to use their Icons as well.

Reply:
fixed

Reply:
I get a type error : 
![image](https://github.com/rango-exchange/rango-client/assets/123297941/3ec11a58-eded-4236-a305-7b139fb6d0fd)

Also your change disappears chain & token logos when they don't have it.
![image](https://github.com/rango-exchange/rango-client/assets/123297941/465a29a2-ea49-4817-8c25-cafff5602f4c)
![image](https://github.com/rango-exchange/rango-client/assets/123297941/7f851bd7-0b73-464b-aed5-f019cf87c56f)

Reply:
Maybe we can separate the Logo component and logo as a string value. checking with typeof is also ok but we should test other ItemPickers that used as well.